### PR TITLE
fix: Change between location, online options etc. moves to top of page

### LINKED ISSUE #6070: Wizard Step 1: Change between location, online etc. moves to top of page
When an organizer creates an event and clicks on the different options like location, online, mixed etc. the page moves to the top. Expected: The page should not move at all.

Screenshot from 2020-12-18 14-03-35

### PR DESCRIPTION
**Pull request by MeghalBisht**
Fixes #6070

#### Short description of what this resolves:
self explanatory

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/forms/wizard/basic-details-step.hbs [hunk 1]
```diff
@@ -42,7 +42,7 @@
   <br>
   <div class="ui secondary pointing menu">
     {{#each this.locationMenuItems as |item|}}
-      <a role="button" href="#" class="{{if (eq item this.selectedLocationType) 'active'}} item" onclick={{action (mut this.selectedLocationType) item}}>
+      <a role="button" href="javascript:void();" class="{{if (eq item this.selectedLocationType) 'active'}} item" onclick={{action (mut this.selectedLocationType) item}}>
```
Comment (daretobedifferent18) [inline app/templates/components/forms/wizard/basic-details-step.hbs:45]: instead use ```{{null}}```

Comment (MeghalBisht) [reply app/templates/components/forms/wizard/basic-details-step.hbs:45]: this would create a dirty url

Comment (MeghalBisht) [reply app/templates/components/forms/wizard/basic-details-step.hbs:45]: null is a better option

**Review by daretobedifferent18 — APPROVED** [review]
Working good for me

**Review by mariobehling — APPROVED** [review]
Nice, that works for me.

**Pull request merged**